packages/daisyui/src/widgets/ToggleWidget/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './ToggleWidget';
+export * from './ToggleWidget';
```

Comment:
Adding blank line at end to remove the red circled dash
```suggestion
export * from './ToggleWidget';

```